code-challenges.test.js

```diff
@@ -15,42 +15,116 @@
 
-const fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
-// Expected output: [-823, 7, 23]
+//************************* PSEUDO CODE *************************//
+// create a function
+// must take in an array as a parameter
+// create a new variable
+// iterate through the array and filter by using higher order function .filter method
+// return the data that's only numbers by using "typeof" method and setting it strictly equals to the value type of "number"
+// use the logical operator of "&&" to strictly return numbers that are divisible by 2 but dont have the remainder of 0, which are odd numbers
+// return the new variable that contains these values and use the .sort() method and sort the values from least to greatest
+//***************************************************************//
 
+  const oddSort = (array) => {
+    let newNum = array.filter(value => {
+     return typeof value === "number" && value % 2 !== 0
+    })
+    return newNum.sort((a,b) => a-b)
+  }
 
-// b) Create the function that makes the test pass.
 
 
-// --------------------3) Create a function that takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.
 
-// a) Create a test with expect statements for each of the variables provided.
 
-const numbersToAdd1 = [2, 4, 45, 9]
-// Excpected output: [2, 6, 51, 60]
 
-const numbersToAdd2 = [0, 7, -8, 12]
-// Expected output: [0, 7, -1, 11]
+// --------------------3) Create a function that takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.
 
-const numbersToAdd3 = []
-// Expected output: []
+// a) Create a test with expect statements for each of the variables provided.
 
+describe("sumArr", () => {
+  it("takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.", () => {
+    const numbersToAdd1 = [2, 4, 45, 9]
+    // Excpected output: [2, 6, 51, 60]
+    const numbersToAdd2 = [0, 7, -8, 12]
+    // Expected output: [0, 7, -1, 11]
+    const numbersToAdd3 = []
+    // Expected output: []
+    expect(sumArr(numbersToAdd1)).toEqual([2, 6, 51, 60])
+    expect(sumArr(numbersToAdd2)).toEqual([0, 7, -1, 11])
+    expect(sumArr(numbersToAdd3)).toEqual([])
+  })
+})
 
 // b) Create the function that makes the test pass.
+
+//************************* PSEUDO CODE *************************//
+// create a function
+// must take in an array as a parameter
+// create a new variable and set it to 0
+// iterate through the array and by using .map() higher order function since it will be return the same amount of values
+// set the new variable created to add to every index of the array and return the sum
+// return the new variable created that contains the new added values
+//***************************************************************//
+
+const sumArr = (array) => {
+  let newSum = 0
+  return array.map(value => {
```

Comment:
You have mastered this technique. I know you can do it here too. One-liner, please?